sql/dql/basic-queries/where.md

```diff
@@ -31,38 +31,59 @@ aspects:
 ---
 ## Content
 
-When added to an SQL `SELECT` statement, `WHERE` allows us to select rows in a table by specifying rules. The `WHERE` keyword can also be used to insert, update and delete data from tables.
+When added to an SQL `SELECT` statement, `WHERE` allows us to select rows in a table by specifying rules. 
 
 You can write a conditional statement after `WHERE` and, for each row in the table, if the conditional statement evaluates to true, the row will be included in the query result. If not, it will not be included in the result.
 
-SQL supports logical operations and comparison operators, such as `=`, `<>`, `>`, `>=`, `<`, `<=`, `AND`, `OR`, `NOT`, `IN`, `BETWEEN`, `LIKE`, `IS NULL` or `IS NOT NULL`. Together, these predicates allow us to define complex rules for extracting just the data that we want from an SQL database. However, if we're not careful, we can easily miss some edge case here, and introduce bugs.
+SQL supports both comparison operators (`=`, `<>`, `>`, `>=`, `<`, `<=`) and logical operations (`AND`, `OR`, `NOT`, `IN`, `BETWEEN`, `LIKE`, `IS NULL`, `IS NOT NULL`). Together, these predicates allow us to define complex rules for extracting just the data that we want from an SQL database. However, if we're not careful, we can easily miss some edge case here, and introduce bugs.
```

Comment:
no need to enumerate all of them 

comparison operators such as `=` or `>=` and logical operators like `AND`, `NOT` or `LIKE` ...

Reply:
Changed phrase to:
>SQL supports both comparison operators (such as `=` and `>=`) and logical operations (such as `AND`, `NOT` or `LIKE`).